### PR TITLE
Adding and Updating code

### PR DESCRIPTION
**Pull request by DylanD16**
The warn commands is missing the permissions decorator so I have added it.

### CODE REVIEW

File: cogs/staff_mod.py [hunk 1]
```diff
@@ -82,6 +82,7 @@ async def case(self, ctx, number: int):
         await ctx.send(embed=embed)
 
     @commands.command()
+    @commands.has_permissions(kick_members=True)
```
Comment (GavynStanley) [inline cogs/staff_mod.py:85]: Following the syntax of the other commands, please make this above the commands decorator

Comment (GavynStanley) [reply cogs/staff_mod.py:85]: And while you're at it, will you add a permissions decorator, with manage_messages to the purge and case commands, pls thx

Comment (DylanD16) [reply cogs/staff_mod.py:85]: Done

**Comment by DylanD16** [comment]
I will push those in a bit when I push the mute command and some others I have made.

**Comment by GavynStanley** [comment]
Ok, when you do push those, push it to this, and rename the PR

**Pull request closed without merging**